plenum/test/input_validation/fields_validation/test_ledger_id_field.py

```diff
@@ -13,15 +14,3 @@ def test_valid_ledger_id():
 def test_invalid_ledger_id():
     not_existing_ledger = 100
     assert validator.validate(not_existing_ledger)
-
-
-def test_plugin_ledger_ids():
```

Comment:
Why is it removed? Do we have another test to test this?

Reply:
I changed `LedgerId` to initialise fields on object creation rather than class declaration